### ISSUE TITLE
Make consecutive kills of the same boss less rewarding

### ISSUE THREAD

**Issue by Kookiecraft** [IdleLands/IdleLands#721]
To discourage drunk walking on top of a boss for days on end.

Some ideas:
1. Lower drop rate of all items/collectibles to zero.
2. Gradually decrease drop rate to zero.
3. Worsen the item pool.
3. Prevent boss kill statistics from incrementing on consecutive kills.
4. Maybe make it require a boss on a different map to be killed to break the consecutive penalty? Think Lich Mansion.
5. Add a cooldown of some sort to reset the consecutive kill counter/penalty after a long period of time (24hr?) has passed without killing any other bosses.
6. Allow each player to have their own boss cooldown timers instead of having one global timer for the entire playerbase per boss. All this would do is allow every player to drunk walk on top of the same boss for loot...

**Comment by seiyria** [comment]
I'm strongly in favor if most of these.

**Comment by Kookiecraft** [comment]
Adding another idea:

8. Increase boss stats after each consecutive kill. Resets to normal after the consecutive chain breaks. I doubt that this would be very effective considering how weak many bosses are, but it would have a lore reason (bosses learn to adapt to the same person killing them over and over).

**Comment by JukeDragh** [comment]
I like 8. If any of the reduce chance ones or other ones are added though, a few rarities might need to be bumped i.e. Gorgon Snake.

**Comment by Aviarium** [comment]
And another one:

9. Flag every Boss with the players name of its last killer. When that player tries to fight the boss again, dont let the fighting event trigger. so after killing a boss, someone else has to kill it, before you are able to fight it again. (#755 was that idea)

Who killed it last could be visible on mouse-over, so for very tough ones like the dragons it is kind of a trophy to have the last kill

**Comment by Farflier** [comment]
A problem I see with scaling down drop rates/stopping stat increments is that it wouldn't prevent trolling, or people randomly getting stuck in a boss area frequently revisiting it. In favor of adding player cooldowns, or Avi's #9 there -- but in combination with #6 to allow another kill in the event nobody else has killed it in 24h, or whatever other timeframe.

**Comment by Aviarium** [comment]
Now we got a few nice solutions, but did not get further on how it should be implemented. Here is a strawpoll to see whats favored by most people. Please take part in it. After a few people contributed to the poll, its up to Sei to decide what should be done.
[Strawpoll](https://strawpoll.de/sweb32r)

And if someone is able to code it afterwards, please assign to the issue, thanks :)

**Comment by kinderghast** [comment]
Currently in game camping Basilisk for the [Gorgon Snake] collectible - literally the only boss i need to do this for, and I believe (from what i could find searching all spells) that Summon (dSummon) is the only spell with such a feature.

I actually really like the flavour that having two extra summons available for those with the required collectible brings.

[Undead Dragon Scale] is fine as it can be picked up each Ascension, could [Gorgon Snake] be moved to an in world item instead of a drop?

Other ideas for alternate requirements to enable the extra summons (or for other spell flavour enhancements for other classes)

- Specific Boss(es) killed that Ascension
- Profession related statistic reached (ie 50k Necromancer Steps, then 100k Necromancer Steps) 
- Achievement related unlock ( Professional: Necromancer Tier 1  = dracolich, Professional: Necromancer Tier 2 = demogorgon)

**Comment by seiyria** [comment]
I agree that boss farming is a problem, however, the idea to make this a random drop was due to how insanely overpowered the demogorgon is compared to other pets.

We currently don't track boss kills per ascension, and I'm not sure I'd want to move to an achievement or stat, because we do have lots of other skill-collectible-unlocks for other classes.

If we were to move this collectible out, it would be to a very remote and tough to get area, probably locked behind those achievements + some.

**Comment by kinderghast** [comment]
Hard to reach area is a good idea, it definitely should be a reward for perseverance and commitment to the Necromantic arts.
Is there a common drop table for high level bosses - what if the dragons in Mythos -3 all had a low chance to drop it? Or perhaps any boss could have a low chance to drop it, but only if the player is a High Level (+250) Necromancer when they kill them.
